packages/perspective-jupyterlab/src/ts/psp_widget.ts

```diff
@@ -152,14 +152,14 @@ export class PerspectiveWidget extends Widget {
      * @param data
      */
     _update(data: TableData): void {
-        this.viewer.update(data);
+        this.viewer.table.update(data);
```

Comment:
`TypeError: this.viewer.table is undefined`

Reply:
this introduces a bug as `table` isn't guaranteed to exist on the viewer. I don't think this was checked?